stdlib/contextlib.pyi

```diff
@@ -80,7 +81,7 @@ class ExitStack(ContextManager[ExitStack]):
     def callback(self, callback: Callable[..., Any], *args: Any, **kwds: Any) -> Callable[..., Any]: ...
     def pop_all(self: _U) -> _U: ...
     def close(self) -> None: ...
-    def __enter__(self: _U) -> _U: ...
+    def __enter__(self: Self) -> Self: ...
```

Comment:
does this need a change in `__aenter__` on AsyncExitStack?

Reply:
I created #5724